lib/uff_db_loader.rb

```diff
@@ -4,12 +4,11 @@
 require "configuration"
 
 module UffDbLoader
-  RAILS_ROOT = Dir.pwd
-  APP_NAME = RAILS_ROOT.split('/').last
-  DUMP_DIRECTORY = File.join(RAILS_ROOT, 'dumps')
-  DATABASE_CONFIG_FILE = File.join(RAILS_ROOT, 'config', 'database.yml').to_s
+  APP_NAME = (Rails.application.class.respond_to?(:parent_name) ? Rails.application.class.parent_name : Rails.application.class.module_parent_name).downcase
+  DUMP_DIRECTORY = Rails.root.join('dumps')
```

Comment:
Did you test this? When I was working on it `Rails.root` was `nil`, because the gem was loaded too early.

Reply:
Doesn't work in the tests too, so need to find a way. However, the old way of `APP_NAME` will fall apart if the directory name is not the app name, for whatever reason.